packages/clerk-js/src/core/signals.ts

```diff
@@ -52,30 +52,38 @@ function errorsToParsedErrors(error: unknown): Errors {
       captcha: null,
       legalAccepted: null,
     },
-    raw: [],
-    global: [],
+    raw: null,
```

Comment:
I hate that this is the case, but we need to update the default errors in the state proxy in clerk-react as well 😭